appeals/web/src/server/lib/page-components/boolean.js

```diff
@@ -0,0 +1,49 @@
+import { convertFromBooleanToYesNo } from '#lib/boolean-formatter.js';
+import { mapActionComponent } from '#lib/mappers/component-permissions.mapper.js';
+import { permissionNames } from '#environment/permissions.js';
+
+/**
+ *
+ * @param {Object} options
+ * @param {string} options.id
+ * @param {string} options.text
+ * @param {boolean|null} [options.value]
+ * @param {string} [options.defaultText]
+ * @param {string} options.link
+ * @param {boolean} [options.addCyAttribute]
+ * @param {import("express-session").Session & Partial<import("express-session").SessionData>} options.session
+ * @returns {Instructions}
+ */
+export function booleanDisplayField({
```

Comment:
Could be more descriptively named, ideally the name would include `instruction` as that's what's being returned

Reply:
Gone with `booleanDisplayInstruction`